paramnb/__init__.py

```diff
@@ -260,7 +260,7 @@ def path_change_event(event):
                 selector.options=named_objs(p_obj.get_range().items())
 
                 if p_obj.objects and not self.p.button:
-                    self.execute(None)
```

Comment:
I'm not sure certain what the intention of ``None`` was here. Should I have changed it to ``{}`` instead?